build.gradle

```diff
@@ -22,9 +18,13 @@ sourceCompatibility = '1.8'
 repositories {
     mavenLocal()
     maven {
-        name = "GT forge Mirror"
-        url = "https://gregtech.overminddl1.com"
+        name = "forge"
+        url = "http://files.minecraftforge.net/maven"
```

Comment:
Isn't it `https://maven.minecraftforge.net`?

Reply:
good catch!